xcube_cci/normalize.py

```diff
@@ -162,4 +164,34 @@ def _normalize_missing_time(ds: xr.Dataset) -> xr.Dataset:
         ds.coords['time_bnds'].attrs['standard_name'] = 'time'
         ds.coords['time_bnds'].encoding['units'] = 'days since 1970-01-01'
 
-    return ds
\ No newline at end of file
+    return ds
+
+
+def normalize_dims_description(dims: dict) -> dict:
+    if 'latitude' in dims:
+        dims['lat'] = dims.pop('latitude')
+    if 'longitude' in dims:
+        dims['lon'] = dims.pop('longitude')
+    if 'latitude_centers' in dims:
+        dims['lat'] = dims.pop('latitude_centers')
+        dims['lon'] = dims['lat'] * 2
```

Comment:
I dont understand this line

Reply:
dims contains integers here. 'latitude_centers' is supposed to have a number of latitude_centers, say, 18 for a 10° resolution. As we will have the same resolution for longitudes, this will be twice the amount (36 for a 10° resolution).
Whenever cate can deal with different resolutions for lat and lon we should just have dims['lon'] = 1 there.